__DEFINES/setup.dm

```diff
@@ -788,6 +788,8 @@ SEE_PIXELS	256
 // Special 'weapons', used in damage procs
 #define WPN_HIGH_BODY_TEMP "High Body Temperature"
 #define WPN_LOW_BODY_TEMP  "Low Body Temperature"
+#define INTERNAL "internal"
```

Comment:
Change these to `RADIATION_INTERNAL` and `RADIATION_EXTERNAL`, or something like that. I would have no idea what these were for just looking at this file.